site/ja/tutorials/images/classification.ipynb

```diff
@@ -0,0 +1,1364 @@
+    "\n",
+    "`tf.keras` においては、このデータ拡張を `ImageDataGenerator` クラスを使用して実装します。データセットに対するさまざまな変換を指定することによって、学習プロセス中にそれが適用されます。"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "2uJ1G030VrWz"
+   },
+   "source": [
+    "### データの拡張と可視化"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "hvX7hHlgVrW0"
+   },
+   "source": [
+    "最初に、ランダムな水平反転による拡張をデータセットに適用し、それぞれの画像が変換後にどのように見えるかを確認します。"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "rlVj6VqaVrW0"
+   },
+   "source": [
+    "### 水平反転の適用"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "xcdvx4TVVrW1"
+   },
+   "source": [
+    " このデータ拡張を適用するためには、 `ImageDataGenerator` クラスの引数として `horizontal_flip` を渡し、 `True`を設定します。"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "Bi1_vHyBVrW2"
+   },
+   "outputs": [],
+   "source": [
+    "image_gen = ImageDataGenerator(rescale=1./255, horizontal_flip=True)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "zvwqmefgVrW3"
+   },
+   "outputs": [],
+   "source": [
+    "train_data_gen = image_gen.flow_from_directory(batch_size=batch_size,\n",
+    "                                               directory=train_dir,\n",
+    "                                               shuffle=True,\n",
+    "                                               target_size=(IMG_HEIGHT, IMG_WIDTH))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "zJpRSxJ-VrW7"
+   },
+   "source": [
+    "学習サンプルから1つのサンプル画像を取得する作業を5回繰り返して、おなじ画像に5回データ拡張が適用されるようにします。"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "RrKGd_jjVrW7"
+   },
+   "outputs": [],
+   "source": [
+    "augmented_images = [train_data_gen[0][0][0] for i in range(5)]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "EvBZoQ9xVrW9"
+   },
+   "outputs": [],
+   "source": [
+    "# 上で学習用画像の可視化のために定義、使用されたおなじカスタムプロット関数を再利用する\n",
+    "plotImages(augmented_images)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "i7n9xcqCVrXB"
+   },
+   "source": [
+    "### 画像のランダムな回転"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "qXnwkzFuVrXB"
+   },
+   "source": [
+    "回転のデータ拡張を利用して学習用サンプルをランダムに45度回転させてみましょう。"
```

Comment:
[weak suggestion] `回転のデータ拡張を` -> `別のデータ拡張である回転を`

Reply:
[suggestion] `45度` -> `45度まで`

Reply:
> [weak suggestion] 回転のデータ拡張を -> 別のデータ拡張である回転を

I think we don't need to use "別の" or another expression here because I didn't use it at [this](https://github.com/tensorflow/docs/pull/1001/files#diff-61c2c3d8e1751c6c4b6ec66728bbf0b4R955) section which is similar expression because there wasn't such an expression in English version.